web/themes/contrib/civictheme/includes/form_element.inc

```diff
@@ -191,6 +191,8 @@ function civictheme_preprocess_fieldset__form_element__civictheme_field__checkbo
       'id' => Html::cleanCssIdentifier($element['#id'] . '-' . $value),
       'is_checked' => is_array($element['#default_value']) ? in_array($value, $element['#default_value']) : $element['#default_value'] == $value,
       'is_disabled' => isset($element['#attributes']['disabled']),
+      // Checkboxes controls with a group cannot be required by definition.
+      'is_required' => FALSE,
```

Comment:
@AlexSkrypnyk If I am understanding you, you are saying you shouldn't compel a user to check all the checkboxes.

Reply:
I’m saying that currently all of the checkboxes are marked required if the field has required attribute. But that required attribute means “at least one checkbox” is required rather than “all checkboxes are required”. Of course, this is in the context of “checkboxes” element only.